datasets/extract/extract_1000_Genomes_NYGC_30x_GRCh38.py

```diff
@@ -0,0 +1,32 @@
+import hailtop.batch as hb
+
+phased_url_root = "ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/" \
+                  "1000G_2504_high_coverage/working/20201028_3202_phased"
+gt_url_root = "ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/" \
+              "1000G_2504_high_coverage/working/20201028_3202_raw_GT_with_annot"
+
+backend = hb.ServiceBackend()
```

Comment:
This is awesome! Can we set an explicit billing project here? I'd like to use hail-datasets-api for now on.

Reply:
Sounds good! Took care of that